lib/mozart_fetcher/http_client.ex

```diff
@@ -25,8 +25,9 @@ defmodule HTTPClient do
         |> log_errors_and_return()
       end
     rescue
-      ex ->
-        Stump.log(:error, %{message: ex})
+      _ ->
+        ExMetrics.increment("http.component.error")
```

Comment:
could you please add the error you got in the logs

Reply:
just to explain why you did this

Reply:
I have added it to the description.